### PR TITLE
Refactor/time scale pause

### PR DESCRIPTION
**Pull request by m4rtint**

### CODE REVIEW

**Comment by m4rtint** [comment]
[Refactor Pause to use timescale](https://trello.com/c/zWI90QaR/87-refactor-pause-to-use-timescale)

File: Assets/Script/Managers/StateManager.cs [hunk 1]
```diff
@@ -134,6 +106,7 @@ public void PlayGame()
     public void EndGame()
     {
         m_state = State.EndGame;
+        Time.timeScale = 1.0f;
```
Comment (Gursaanj) [inline Assets/Script/Managers/StateManager.cs]: Should we stop time when the game has ended (i.e. timescale = 0.0f)?

Comment (m4rtint) [reply Assets/Script/Managers/StateManager.cs]: Mmmmmm I thought it'll be funnier if we let it bounce around when the game ends when peolle still accelerating

Comment (Gursaanj) [reply Assets/Script/Managers/StateManager.cs]: Oh I see haha fair enough. But if that is the case, wouldnt the timescale already be 1.0 when Endgame is called.  so there is no point of setting the timescale in the method?

Comment (m4rtint) [reply Assets/Script/Managers/StateManager.cs]: It's a precaution

Comment (Gursaanj) [reply Assets/Script/Managers/StateManager.cs]: fair enough

File: Assets/Script/Managers/StateManager.cs [hunk 2]
```diff
@@ -156,34 +129,4 @@ private void Awake()
 
         DontDestroyOnLoad(gameObject);
     }
-
```
Comment (wbeltran777) [inline Assets/Script/Managers/StateManager.cs]: Where are we freezing and unfreezing the rigidbodies now that they're not in statemanager?

Comment (m4rtint) [reply Assets/Script/Managers/StateManager.cs]: No more freezing or refreezing. No need to.

Comment (m4rtint) [reply Assets/Script/Managers/StateManager.cs]: We're just freezing time.

Comment (wbeltran777) [reply Assets/Script/Managers/StateManager.cs]: Groovy

**Pull request merged**